charts/radix-cicd-canary/templates/metrics.yaml

```diff
@@ -24,15 +24,16 @@ metadata:
     {{- toYaml .Values.service.annotations | nindent 4 }}
   {{- end }}
   labels:
-    app: radix-cicd-canary
-  name: radix-cicd-canary
-  namespace: radix-cicd-canary
+    {{- include "cicd-canary.labels" . | nindent 4 }}
+  name: {{ template "cicd-canary.fullname" . }}
+  namespace: {{ .Release.Namespace }}
 spec:
   ports:
   - name: metrics
     port: 5000
     protocol: TCP
     targetPort: metrics
   selector:
-    app: radix-cicd-canary
+    matchLabels:
```

Comment:
selector does not have a matchLabels prop. Labels must be placed directly inside selector.

Tip. You can test the helm chart with --dry.run. Make sure you are in the chart root folder.
`helm install radix-cicd-canary -n radix-cicd-canary . --dry-run`